src/System.Text.Json/tests/System.Text.Json.Tests.csproj

```diff
@@ -23,6 +23,12 @@
     <Compile Include="JsonReaderStateAndOptionsTests.cs" />
     <Compile Include="JsonTestHelper.cs" />
     <Compile Include="JsonWriterOptionsTests.cs" />
+    <Compile Include="NewtonTests\CamelCaseTests.cs" />
```

Comment:
Nit: Could the directory be `NewtonsoftTests` 🙏

Reply:
Sure thing